src/runway/sources/git.py

```diff
@@ -0,0 +1,100 @@
+""" 'Git' type Path Source """
+
+import tempfile
+import shutil
+import subprocess
+
+import os
+import sys
+import six
+import logging
+
+from .source import Source
+
+LOGGER = logging.getLogger('runway')
+
+
+class Git(Source):
+    """ Git type Path Source """
+
+    def __init__(self, config, cache_dir=None):
+        LOGGER.info(config)
+        self.config = config
+        self.cache_dir = cache_dir
```

Comment:
This looks like something that could belong in the parent class.

Reply:
Good call, will move it up